runtime/criusupport/j9criu.tdf

```diff
@@ -51,5 +51,5 @@ TraceException=Trc_CRIU_setupJNIFieldIDsAndCRIUAPI_null_exception_class Obsolete
 TraceException=Trc_CRIU_setupJNIFieldIDsAndCRIUAPI_load_criu_failure Obsolete Overhead=1 Level=1 Template="setupJNIFieldIDsAndCRIUAPI() The JVM attempted to load libcriu.so but was unable to: %zi"
 TraceException=Trc_CRIU_setupJNIFieldIDsAndCRIUAPI_not_find_criu_methods Obsolete Overhead=1 Level=1 Template="setupJNIFieldIDsAndCRIUAPI() The JVM could not find critical criu functions in libcriu.so: %zi"
 
-TraceEvent=Trc_CRIU_after_criu_dump Obsolete Overhead=1 Level=2 Template="After checkpoint criu_dump(), criuRestoreNanoTimeMonotonic (%lld), criuRestoreNanoUTCTime (%llu), lastRestoreTimeMillis (%lld)"
+TraceEvent=Trc_CRIU_after_criu_dump Obsolete Overhead=1 Level=2 Template="After checkpoint criu_dump(), criuRestoreNanoTimeMonotonic (%lld), criuRestoreNanoUTCTime (%llu), lastRestoreTimeInNanoseconds (%llu)"
```

Comment:
Generally speaking, dont change existing tracepoints. Also, this one is obsolete so it will not have an effect